.eslintrc.js

```diff
@@ -80,6 +80,14 @@ module.exports = {
     'simple-import-sort/imports': 'error',
     'simple-import-sort/exports': 'error',
   },
+  overrides: [
+    {
+      files: 'bin/**',
+      rules: {
+        'no-var': '0',
```

Comment:
Minor, can we change this to the string version of this error code?
```suggestion
        'no-var': 'off',
```